resources/views/guest/welcome.blade.php

```diff
@@ -14,4 +14,9 @@
         </p>
     </div>
 </div>
-@endsection
+<div>
+    <div class="col-lg-6 px-0">
+        @livewire('public-home.today-lecturing-schedules')
+    </div>
+</div>
+@endsection
```

Comment:
Tambahkan empty line ending
```suggestion
@endforelse

```